### PR TITLE
kubel fetch api resource list add kubel-context

### PR DESCRIPTION
**Pull request by baron42bba**
add kubel-context to kubel--fetch-api-resource-list

### CODE REVIEW

File: kubel.el [hunk 1]
```diff
@@ -734,7 +734,8 @@ ARGS is the arguments list from transient."
 
 (defun kubel--fetch-api-resource-list ()
   "Fetch the API resource list."
-  (split-string (kubel--exec-to-string "kubectl api-resources -o name --no-headers=true") "\n"))
+  (split-string (kubel--exec-to-string
+		 ( format "kubectl --context %s api-resources -o name --no-headers=true" kubel-context )) "\n"))
```
Comment (abrochard) [inline kubel.el:738]: is there an extra space in after `(` and before `)`?

**Comment by abrochard** [comment]
Thank you for submitting this. I left a quick comment. Should be able to merge soon

File: kubel.el (hunk 1 above)
Comment (baron42bba) [reply kubel.el]: Eagle eye!
Blank is now removed.

**Review by abrochard — APPROVED** [review]
Thanks!

**Pull request merged**